core/modules/layout/layout.module

```diff
@@ -2055,6 +2055,100 @@ function layout_layout_info() {
   return $layouts;
 }
 
+/**
+ * Generate an image to be used as a layout template thumbnail.
```

Comment:
This should be in a separate PR. I think this may be part of https://github.com/backdrop/backdrop-issues/issues/4104 ?

Reply:
You're right; I was merging changes from both issues, so to file combined PRs. Fixed.